plugins/inputs/http_listener_v2/http_listener_v2.go

```diff
@@ -186,10 +193,20 @@ func (h *HTTPListenerV2) Stop() {
 	h.wg.Wait()
 }
 
+func (h *HTTPListenerV2) containsPath(needle string) bool {
+	for _, v := range h.Paths {
+		if v == needle {
+			return true
+		}
+	}
+
+	return false
+}
+
```

Comment:
Please directly use the functionality from `"github.com/influxdata/telegraf/internal/choice"` as
```golang
choice.Contains(needle, h.Paths)
```